package.json

```diff
@@ -23,7 +23,7 @@
     }
   ],
   "main": "./index",
-  "module": "./module/index",
+  "module": "./index",
```

Comment:
we don't ship an ESM build yet so it's better to just remove this field (`module` in pjson make tools think this is the ESM entrypoint, we don't want CJS be treated as ESM).